app/Http/Controllers/Admin/DiagramController.php

```diff
@@ -64,15 +73,57 @@ public function saveWorkspace(Request $request)
         return redirect()->route('generate', ['id' => $workspace->id]);
     }
 
-    public function generateDiagram(Request $request, $idWorkspace)
+    public function getAvailableSeatsByDate(Request $request, $id)
     {
-        $workspace = $this->workspace->findOrFail($idWorkspace);
-        $renderSeat = $this->renderSeat($workspace);
-        $locations = $workspace->locations;
-        $listProgram = $this->programRepository->listProgramArray();
-        $listPosition = $this->positionRepository->listpositionArray();
+        $user = Auth::user();
+        $firstDay = date('Y') . '-' . date('m') . '-01';
+        $lastDay = date('Y') . '-' . date('m') . '-' . date('t');
+        $workSchedules = WorkSchedule::where('user_id', $user->id)
+                            ->with(['seats' => function ($query) {
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Controllers/Admin/DiagramController.php</br>**Line**: 82</br>Each line in an array declaration must end in a comma